tests/integration/fixtures/users.const.ts

```diff
@@ -0,0 +1,25 @@
+import { UserRole } from '../../../services/API-service/src/api/user/user-role.enum';
+import { UserStatus } from '../../../services/API-service/src/api/user/user-status.enum';
+
+// REFACTOR: it will happen a lot throughout these integration tests to use DTO's from the API-service. Is it fine to import them like this?
```

Comment:
I suggest we consider the service and test suite as independent apps. My counter-argument to my own suggestion is that it leads to duplicate enums and types.

This is an existing problem between the dashboard and API service. I believe the decoupling is worth the duplication costs. We should explore a _proper_ way to resolve this when we have capacity.